AGENTS.md

```diff
@@ -77,6 +78,7 @@ The repository is organized as a monorepo with the following packages:
  -  *packages/sqlite/*: SQLite driver (@fedify/sqlite)
  -  *packages/sveltekit/*: SvelteKit integration (@fedify/sveltekit)
  -  *packages/testing/*: Testing utilities (@fedify/testing)
+ -  *packages/vocab-runtime/*: Runtime utilities and types (@fedify/vocab-runtime)
```

Comment:
This does not reflect the actual directory structure: no *packages/vocab-runtime/*, but *packages/vocab/runtime*. However, I personally prefer *packages/vocab-runtime/* over *packages/vocab/runtime*, because it's handier to deal with them with automated scripts.